chronicle/ViewModels/EnrollmentViewModel.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  EnrollmentViewModel.swift
+//  EnrollmentViewModel
+//
+//  Created by Alfonce Nzioka on 8/11/21.
+//
+
+import Foundation
+
+/// Grants view access to the Enrollment model.  Keeps track of input values and errors
+class EnrollmentViewModel: ObservableObject {
```

Comment:
What is a "ViewModel"? Consider renaming.

Reply:
The class primarily handles state management, thus keeping views as stateless as possible. Would prefer to keep the name as it alludes to the MVVM architectural paradigm underlying the development of the app